tests/redirects/base.py

```diff
@@ -131,8 +131,11 @@ def assert_valid_url(
     :param final_status_code: Expected status code after following any redirects.
     """
     kwargs = {"allow_redirects": follow_redirects}
-    if req_headers:
+    if req_headers is None:
+        kwargs["headers"] = {"Accept-language": "en"}
```

Comment:
Was this added to avoid having to patch a ton of existing tests?

Reply:
Yes, for the redirects it made sense to me as a default.